pkg/runtime/runtime.go

```diff
@@ -163,11 +164,16 @@ func newDaprRuntime(ctx context.Context,
 	resiliencyProvider resiliency.Provider,
 ) (*DaprRuntime, error) {
 	compStore := compstore.New()
+	wasmStrictSandbox := (*bool)(nil)
```

Comment:
Why is this a pointer? I don't think the "nil" case matters here:

CRD can force the strict sandbox to be enabled, but it cannot force it to be disabled. If for a specific component the developers/ops teams want it to be enabled, it should be enabled.

Reply:
Done. changed to bool.